setup.py

```diff
@@ -9,6 +9,7 @@
 install_requires = [
     'jinja2',
     'lxml',
+    'requests',
```

Comment:
This is likely a `collation_requires` value, but it won't hurt to include it since most other applications use it anyhow.